### ISSUE TITLE
Firefox name WAY to long 

### ISSUE THREAD

**Issue by Dutchglory** [johannhof/emoji-helper#104]
with Firefox the name is *WAY* to long,  please set a short name... 
 
thanks

**Comment by rugk** [comment]
Ah, you mean that in the overflow menu:
![grafik](https://user-images.githubusercontent.com/11966684/37529744-2ef70822-2938-11e8-85d1-0f80a74fd38e.png)

Indeed, that's a bit too long.

**Comment by Prometheos2** [comment]
Weirdly enough, it doesn't seem to be the case on [Mozilla's add-on website](https://addons.mozilla.org/fr/firefox/addon/emoji-cheatsheet/).

I will see if I can identify the problem.

**Comment by rugk** [comment]
As I've now also made [a similar add-on](https://addons.mozilla.org/firefox/addon/awesome-emoji-picker/) (*) I can explain it:  
* You can set a custom title for your add-on on AMO AFAIK.
* the name in the screenshot of the button menu is actually [set here in the `manifest.json` file](https://github.com/johannhof/emoji-helper/blob/1796ad7944a9712482b984d10abbc8a214c4a5bf/manifest.json#L4)

\* uhm, yes, shameless competition ad here :stuck_out_tongue_winking_eye:

**Comment by Prometheos2** [comment]
So, @rugk, you want to do the PR, or should I do it?

Thank you for the info.

**Comment by rugk** [comment]
Feel free to do so, I am just not sure whether this here is really maintained anymore. (opened https://github.com/johannhof/emoji-helper/issues/127 for that now)

**Comment by Prometheos2** [comment]
Yeah, I thought the same; seeing how he doesn't react to mention doesn't help.
